core/modules/ccontrol/UserQueryDrop.cc

```diff
@@ -81,11 +76,11 @@ void UserQueryDrop::submit() {
     // Just mark this db/table in CSS with special status, watcher
     // will take care of the actual delete process
 
-    LOGF(getLogger(), LOG_LVL_INFO, "going to drop db or table - %s.%s" % _dbName % _tableName);
+    LOGS(_log, LOG_LVL_DEBUG, "About to drop: " << _dbName <<  "." << _tableName);
```

Comment:
I would probably keep this at INFO level, this sort of queries should be rare and they are destructive, I'd like to see a record of when/what caused it.